.travis.yml

```diff
@@ -12,3 +12,7 @@ cache: npm
 
 before_install:
   - sh preinstall.sh
+
+script:
+  - npm run build
+  - npm test
```

Comment:
This won't do anything so you should just remove it

Reply:
Deleted commit